event_project/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Module name
+===========
+
+This module allows you to create events with assigned a project.
+You can make project template and assign this template to new event,
+this, create and calculate the project tasks dates.
+And if the event date start modify, recalculate tasks dates.
+
+
+Installation
+============
+
+To install this module, you need to:
+
+* event
+* project
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+-First step, you create a project template.
+-Second step, you indicate in to task of template the previous days to begin task.
+-And Last step, you create a new event and select project template.
+Automatically, will be create a new related project with event.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Antiun Ingenieria S.L. (Madrid, Spain, http://www.antiun.com)
```

Comment:
Companies are not contributors, unless it is a generic reference (which is not the case).